js/client-battle-tooltips.js

```diff
@@ -343,7 +348,21 @@ var BattleTooltips = (function () {
 				text += '&nbsp;SpA /&nbsp;' + myPokemon.stats['spd'] + '&nbsp;SpD /&nbsp;';
 			}
 			text += myPokemon.stats['spe'] + '&nbsp;Spe</p>';
-			if (isActive) text += '<p class="section">Opponent sees:</p>';
+			if (isActive) {
+				pokemon.stats = myPokemon.stats;
+				pokemon.ability = ability;
+				pokemon.item = item;
+				pokemon.hp = myPokemon.hp;
+				pokemon.maxhp = myPokemon.maxhp;
+				pokemon.unburden = myPokemon.unburden;
```

Comment:
The option was to either modify the pokemon or add the boosts and a few other things to the myPokemon. I tried it both ways, and both ways I had to add things.